setup.py

```diff
@@ -22,7 +22,7 @@ def read(fname):
 
 setup(
     name='ecs-cluster',
-    version='1.5.1',
+    version='1.6.0',
```

Comment:
What do you think of using `1.5.2`?
Can you see any possible incompatibilities with builds using `1.5.1`?

Reply:
Good point.  I initially changed a parameter in the get_latest_task_definition function but I have since reverted that change, so it should remain compatible